tldraw/packages/react/src/hooks/useGestureEvents.ts

```diff
@@ -99,6 +99,7 @@ export function useGestureEvents(ref: React.RefObject<HTMLDivElement>) {
         max: TLViewport.maxZoom,
         min: TLViewport.minZoom,
       }),
+      modifierKey: isDarwin() ? 'metaKey' : 'ctrlKey',
```

Comment:
The `modifierKey` of the pinch event is handled by the [use-gesture library config](https://github.com/pmndrs/use-gesture/blob/main/packages/core/src/types/config.ts#L101), and only accepts a [single mod key](https://github.com/pmndrs/use-gesture/blob/main/packages/core/src/types/config.ts#L99). I tried to manually transition to `select.pinching` when the `metaKey` is pressed, but I didn't manage to make it work properly.

Reply:
When this option is turned on, I am no longer able to pinch the canvas without `metaKey` modifier - is this intended?

Reply:
It shouldn't be, The default mod key is `ctrl` [according to their docs](https://github.com/pmndrs/use-gesture/blob/main/packages/core/src/types/config.ts#L121). I'll revert this, until we find a better solution.

Reply:
Reverted.